src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -47,31 +41,35 @@ def schedule_auto_transition_new() -> None:
 )  # type: ignore
 def auto_transition_issues_new_to_ongoing(
     project_id: int,
-    date_added_lte: int,
-    date_added_gte: Optional[int] = None,
+    first_seen_lte: int,
+    first_seen_gte: Optional[int] = None,
```

Comment:
Not totally sure if using `first_seen` is the move here. 

Issues that go through a `new` -> `resolved` <-> `regressed` loop will become `ongoing` 14 days after the first event that groups to this issue was observed. 
```
days:
0  1  2  3  4  5  6  7  8  9  10  11  12  13  14
----------------------------------------------------> issue state
^              ^                           ^     ^
new         resolved                regressed    ongoing

```

An alternative is we can use `GroupHistory` to determine when the Group status changed and use that as the 'last active' date to count down and automatically transition the state to ongoing.